SU-SWS/first_step_subtheme
# TSUE-96: Algolia Search Resources

# READY FOR REVIEW

# Summary
- Algolia search PDB for Techsource resources

# Review By (Date)
- Retro/Never?

# Steps to test

1. Navigate to: https://sandboxtechsource.sites.stanford.edu/algolia-resources-filter-demo-page
2. Ensure search with interface loads
3. Click on a facet or two to filter them 
4. Review results 
5. Copy URL to clipboard, close browser window, open new window, paste URL, and return to previous state
6. Toggle the sorting A-Z, Z-A and the with relevance. 
7. Do all the above again but on mobile
